### PR TITLE
Add all Azure Functions x Azure Developer CLI samples are added to the gallery

### LINKED ISSUE #101: Add all Azure Functions x Azure Developer CLI samples are added to the gallery
https://github.com/Azure-Samples?q=function+azd-templates&type=all&language=&sort=

### PR DESCRIPTION
**Pull request by hemarina**
fix #101

### CODE REVIEW

**Review by rajeshkamal5050 — APPROVED** [review]

**Comment by savannahostrowski** [comment]
@paulyuk in case you want to take a look!

**Review by paulyuk — APPROVED** [review]
LGTM

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -309,6 +319,26 @@ const Users: User[] = [
     source: 'https://github.com/tonybaloney/simple-flask-azd',
     tags: ['python'],
   },
+  {
+    title: 'Function C# AI Textsummarize',
```
Comment (paulyuk) [inline website/src/data/users.tsx]:
```suggestion
    title: 'Function App - C# AI Text Summarize',
```

File: website/src/data/users.tsx [hunk 2]
```diff
@@ -309,6 +319,26 @@ const Users: User[] = [
...
+    description:
+      'This sample shows how to take text documents as a input via BlobTrigger, does Text Summarization processing using the AI Congnitive Language service, and then outputs to another text document using BlobOutput binding.',
+      preview: require('./images/test.png'),
+    website: 'https://github.com/paulyuk',
+    author: 'Paul Yuknewicz',
+    source: 'https://github.com/Azure-Samples/function-csharp-ai-textsummarize',
+    tags: ['functions','csharp','dotnet','azureai', 'bicep'],
+  },
+  {
+    title: 'Function Python AI Textsummarize',
```
Comment (paulyuk) [inline website/src/data/users.tsx]:
```suggestion
    title: 'Function App - Python AI Text Summarize',
```

**Review by paulyuk — CHANGES_REQUESTED** [review]
Asking for some title tweaks

**Pull request merged**